.circleci/scripts/install-cypress.sh

```diff
@@ -4,7 +4,7 @@ echo "Installing Cypress..."
 
 # Install Cypress dependencies
 sudo apt-get update
-sudo apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb -y
+sudo NEEDRESTART_MODE=a apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb -y
```

Comment:
Not sure why this is suddenly a problem (a service dependency has probably been recently updated?) but there is an interactive popup that is displayed in the terminal asking which services should be restarted. This environment variable tells `apt-get` to restarts the services automatically and prevents the popup.

See output from one of the failing builds here: https://app.circleci.com/pipelines/github/ministryofjustice/bichard7-next-user-service/1643/workflows/41339a27-30b1-4cbe-ae10-19e610b41bd1/jobs/6999?invite=true#step-112-237

See here for solutions:
https://askubuntu.com/questions/1367139/apt-get-upgrade-auto-restart-services

Reply:
Nice! I had the same issue in the ui repo and made a change in the config file to fix it: https://github.com/ministryofjustice/bichard7-next-ui/blob/aa78632184114e408fce9a34073ac341e6dd60a0/.circleci/scripts/install-cypress.sh#L5

This env var is much cleaner though 👍